src/sage/groups/perm_gps/permgroup_named.py

```diff
@@ -3435,3 +3437,91 @@ def codegrees(self):
         ret = [self._m * i for i in reversed(range(self._n-1))]
         ret.append((self._n-1)*self._m - self._n)
         return tuple(sorted(ret, reverse=True))
+
+class SmallPermutationGroup(PermutationGroup_generic):
+    r"""
+    A GAP SmallGroup, returned as a permutation group.
+
+    GAP contains a database of groups, each identified by its GAP id.
```

Comment:
GAP contains many databases of groups, with an id. This should clarify which one you are referring to: the small groups library? transitive groups? primitive groups? perfect groups? ...

if it is the small groups library (SGL), the correct name would be more like "small groups id" -- note that Magma uses the exact same ids